src/less/less/ds6/mixins.less

```diff
@@ -91,23 +91,184 @@
     color: @ds6-color-g206-gray;
 }
 
+// Iconography
+//-----------------------------
+
+.dimensions(@width; @height) {
+    height: @height;
+    width: @width;
+}
+
 .background-icon-base() {
     background-repeat: no-repeat;
     display: inline-block;
     vertical-align: middle;
 }
 
+.background-icon-default() {
+    .background-icon-base;
+    height: 1rem;
+    width: 1rem;
+}
+
+.background-icon(@width; @height; @base64) {
+    background-image: url('data:image/svg+xml;base64,@{base64}');
+    background-size: contain;
```

Comment:
can `background-size: contain` be moved to `.background-icon-base() ` mixin since its a common across?

Reply:
Yeah, that probably makes sense. I think I left this in here because I previously had it as `background size: @width @height;` before changing to `contain`.